Src/PerformanceCollector/Shared/QuickPulseTelemetryModule.cs

```diff
@@ -120,6 +124,12 @@ public QuickPulseTelemetryModule()
         public bool DisableTopCpuProcesses { get; set; }
 
         /// <summary>
+        /// Gets the auth API key.
```

Comment:
I don't actually know how to obtain it exactly just yet :) I think we'll post a blog post once everything is finalized. For now, I've added a comment stating that the key is created in the portal and serves the purpose of secure configuration distribution.